### PR TITLE
Update action-setup-pnpm to v3

### PR DESCRIPTION
**Pull request by nicolasgasco**
The aim of this MR is using `wyvox/action-setup-pnpm@v3` in place of `NullVoxPopuli/action-setup-pnpm@v2` in our CI script.

`NullVoxPopuli/action-setup-pnpm` has been [archived](https://github.com/NullVoxPopuli/action-setup-pnpm) and moved to [another repository](https://github.com/wyvox/action-setup-pnpm).

### CODE REVIEW

**Comment by nicolasgasco** [comment]
#web

**Review by poulet42 — COMMENTED** [review]
I think the `no-lockfile` option has been removed

**Comment by nicolasgasco** [comment]
> I think the `no-lockfile` option has been removed

It was @poulet42 , but it was already refactored to `no-lockfile: true` for this project some months ago

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -220,7 +220,7 @@ importers:
         specifier: ^1.1.2
         version: 1.1.2
       '@qonto/ember-lottie':
-        specifier: 0.6.0
+        specifier: 0.6.1
```
Comment (dannycalleri) [inline pnpm-lock.yaml:223]: Is this `@qonto/ember-lottie` update wanted?

Comment (dannycalleri) [reply pnpm-lock.yaml:223]: Oh! Indeed it is 🤔 https://github.com/qonto/ember-lottie/blob/main/test-app/package.json#L40

Alright, nice catch 🙏

**Review by dannycalleri — APPROVED** [review]

**Comment by poulet42** [comment]
> > I think the `no-lockfile` option has been removed
> 
> It was @poulet42 , but it was already refactored to `no-lockfile: true` for this project some months ago

@nicolasgasco  Sorry I don't think I understood your reply 😄 , you can't use 
```
with:
  no-lockfile: true
```
anymore with the new version of this action, you have to rewrite it as 
```
with:
  args: --no-lockfile
```

**Pull request merged**